shared/styles/mixins.scss

```diff
@@ -83,17 +97,69 @@
   padding: 0 $container-gutter-mobile;
```

Comment:
Should this also be long hand?

Reply:
Hmm, I'm not sure. Is there a case where we need to explicitly force it to be `0`? @organdonor47 @mdmagnusson ?

Reply:
@osk @k80 yeh it should i think. no reason why it needs to be 0 top & bottom